### PR TITLE
Add org.overte.Overte

### PR DESCRIPTION
**Pull request by JulianGro**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
Overte is a social-VR platform. This Flatpak specifically includes the client called "interface".

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:** https://github.com/orgs/overte-org/people

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
pr should be made against the `new-pr` branch

File: flathub.json
Comment (hfiguiere) [inline flathub.json:1]: you can remove this file.

File: org.overte.Overte.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: Overte
+    buildsystem: simple
+    cleanup:
+      - squashfs-root
+    sources:
+      - type: extra-data
```
Comment (hfiguiere) [inline org.overte.Overte.yml:43]: why using extra-data?

File: org.overte.Overte.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        only-arches:
+          - x86_64
+        url: https://public.overte.org/build/overte/release/2024.07.1/Overte-2024.07.1-x86_64.AppImage
```
Comment (hfiguiere) [inline org.overte.Overte.yml:46]: no https download?

**Comment by JulianGro** [comment]
> pr should be made against the `new-pr` branch

@hfiguiere how do I do that? GitHub allowed me to target it, but I couldn't open a PR at that point.
I just tried retargetting, but it automatically closed the PR.

**Comment by JulianGro** [comment]
.

File: org.overte.Overte.yml (hunk 1 above)
Comment (JulianGro) [reply org.overte.Overte.yml:43]: This is an AppImage being extracted. Is there a preferable method?
Maybe the `extra-data` is what calls the required `apply_extra.sh` script?

File: org.overte.Overte.yml (hunk 2 above)
Comment (JulianGro) [reply org.overte.Overte.yml:46]: It is https.
If you mean the aarch64 AppImage underneath, that works in practice but isn't an official release. It is mainly a placeholder until the next release next month.  I can upload it to public.overte.org in a couple of days if necessary, I just don't have access right now.

**Comment by hfiguiere** [comment]
that apply basing the branch off `new-pr`. Which is documented there: https://docs.flathub.org/docs/for-app-authors/submission#submission-pr

**Comment by JulianGro** [comment]
> that apply basing the branch off `new-pr`. Which is documented there: https://docs.flathub.org/docs/for-app-authors/submission#submission-pr

Sorry about that. I just noticed that as well. I will open a new PR in a couple of minutes.

**Comment by JulianGro** [comment]
https://github.com/flathub/flathub/pull/5509

**Pull request closed without merging**